src/BusyIndicator/BusyIndicator.js

```diff
@@ -0,0 +1,63 @@
+import { BUSY_INDICATOR_SIZES } from '../utils/constants';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+
+const BusyIndicator = React.forwardRef(({
+    children,
+    className,
+    show,
+    size,
+    disableStyles,
+    ...props
+}, ref) => {
+
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/icon.css');
+            require('fundamental-styles/dist/busy-indicator.css');
+        }
+    }, []);
+
+    const busyIndicatorClasses = classnames(
+        {
+            [`fd-busy-indicator--${size}`]: !!size
+        },
+        className
+    );
+
+    if (!show)
+        return (<></>);
```

Comment:
Should we do
```jsx
if (!show) {
  return null;
}
```
to match the style of other components in fundamental-react?

Reply:
Added above suggested changes.